tests/unit/stores/ontRoot.spec.js

```diff
@@ -122,7 +122,7 @@ describe('useOntRootStore', () => {
         const rootStore = useRootStore()
         const get = vi.fn()
         get.mockRejectedValue(failedResponse)
-        rootStore.api.v1 = { traction: { ont: { instruments: { get } } } }
+        rootStore.api.v2 = { traction: { ont: { instruments: { get } } } }
```

Comment:
has ont root store changed?

Reply:
No, it hasn't. But it was failing with v1 end point. I might probably need to figure this out with @stevieing